CONTRIBUTING.md

```diff
@@ -1 +1,8 @@
-We require contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+# Contribution guidelines
```

Comment:
I copied the structure of GitHub's atom https://github.com/atom/atom/blob/master/CONTRIBUTING.md :) I think it would be nice to have a top header?